go_modules/lib/dependabot/go_modules/file_updater/go_mod_updater.rb

```diff
@@ -14,9 +14,15 @@ class GoModUpdater
         ENVIRONMENT = { "GOPRIVATE" => "*" }.freeze
 
         RESOLVABILITY_ERROR_REGEXES = [
+          # Private module could not be fetched
           /go: .*: git fetch .*: exit status 128/.freeze,
+          # The checksum in go.sum does not match the dowloaded content
           /verifying .*: checksum mismatch/.freeze,
+          # Private module could not be found during `go mod tidy`
+          /cannot find module providing package .*: exit status 128/m.freeze,
+          # Private module could not be found during `go build`
           /build .*: cannot find module providing package/.freeze,
+          # Package in module was likely renamed or removed
```

Comment:
Scope creep: https://github.com/facebook/ent/pull/660 caused 
```
go: github.com/facebookincubator/ent@v0.4.2: parsing go.mod:\n\tmodule declares its path as: github.com/facebook/ent\n\t        but was required as: github.com/facebookincubator/ent\n
```

Seems related to this block, but not what we're here for. (I was 🤞 it would be the missing reproduction).

Reply:
That would currently bubble up as a not_parseable_error I think, does that seem right?